integration

```diff
@@ -139,6 +139,7 @@
   "claudegel/sinope-gt125",
   "claytonjn/hass-circadian_lighting",
   "ClusterM/skykettle-ha",
+  "CM000n/qss",
```

Comment:
https://github.com/CM000n/qss/blob/20221226-1/hacs.json
Needs a new release.

Reply:
Thanks for the hint!